helm/ovn-kubernetes/values-multi-node-zone.yaml

```diff
@@ -166,3 +166,7 @@ monitoring:
   # in defining that.
   commonServiceMonitorSelectorLabels:
     release: kube-prometheus-stack
+  # -- deploy ServiceMonitors for service discovery using the Prometheus Operator
+  enableServiceMonitor: false
+  # -- deploy PrometheusRules for specific metric collection using the Prometheus Operator
+  enablePrometheusRule: false
```

Comment:
IMO: these values and the related manifests can be removed in this PR, please see https://github.com/ovn-kubernetes/ovn-kubernetes/pull/5510#issuecomment-3347832347